bentoml/saved_bundle/bentoml-init.sh

```diff
@@ -18,13 +18,11 @@ if [ -f ./python_version ]; then
   DESIRED_PY_VERSION=${PY_VERSION_SAVED:0:3} # returns 3.6, 3.7 or 3.8
   CURRENT_PY_VERSION=$(python -c 'import sys; print(f"{sys.version_info.major}.{sys.version_info.minor}")')
 
-  if [[ "$DESIRED_PY_VERSION" == "$CURRENT_PY_VERSION" ]]; then
-    echo "Python Version in docker base image $CURRENT_PY_VERSION matches requirement python=$DESIRED_PY_VERSION. Skipping."
+  if command -v conda >/dev/null 2>&1; then
```

Comment:
This was meant to avoid installing Python - when the user uses a base image with Python pre-installed.  Is the idea that if `conda` command is presented, then re-installing the same python version should be fast?  Does a different `patch` version affects this behavior? For example, if the base image has 3.7.1 but BentoService requires 3.7.4?

Reply:
As I said in the context, we have to call `conda install` to lock the Python version, or it will try to install Python3.9 once `conda install pip` called.